tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+"""PyTest configuration."""
+
+
+def pytest_configure():
+    """Generate SQL files before running tests."""
+    exec(open("script/generate_sql").read())
```

Comment:
This code will have to be at the top-level, not inside a function, in order for it to run when `conftest.py` is imported.

Reply:
I was thinking it would get called automatically: https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_configure But I'll look into it.

Reply:
Ah, wasn't aware of that. Seems like a good approach. It may be getting skipped because it expects to be passed an argument.